tracer/src/Datadog.Trace/DatabaseMonitoring/DatabaseMonitoringPropagator.cs

```diff
@@ -0,0 +1,43 @@
+// <copyright file="DatabaseMonitoringPropagator.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Text;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Propagators;
+using Datadog.Trace.Util;
+
+namespace Datadog.Trace.DatabaseMonitoring
+{
+    internal static class DatabaseMonitoringPropagator
+    {
+        private static string sqlCommentSpanService = "dddbs";
+        private static string sqlCommentRootService = "ddps";
+        private static string sqlCommentVersion = "ddpv";
+        private static string sqlCommentEnv = "dde";
+
+        internal static string PropagateSpanData(DbmPropagationLevel propagationStyle, string configuredServiceName, Span span)
```

Comment:
Since this method takes the DbmPropagationLevel enum, you should handle the case where `propagationStyle == DbmPropagationLevel.Disabled`. In that case you could probably return `string.Empty`

Reply:
I also suggest adding a unit test that passes `DbmPropagationLevel.Disabled` as input